### PR TITLE
Convert: Diff Two Arrays Challenge Solution

### PR DESCRIPTION
**Pull request by abhisekp**
## Algorithm: Diff Two Arrays

Use new [**Challenge Solution Template**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Wiki-Template-Challenge-Solution)
Also add advanced solution

----
If wanted, I can split it into multiple commits for better history.

----
**Please confirm before merging.**  
I will squash locally and push :+1: 

----
> Pending: add code explanations for intermediate and advanced solutions

Help Wanted

Pull Request to my branch WELCOME! :100:

### CODE REVIEW

**Comment by Rafase282** [comment]
Looks good to me.

**Comment by abhisekp** [comment]
@Rafase282 Need help by someone to explain the intermediate and advanced codes. Both are quite similar.

**Comment by alayek** [comment]
@abhisekp are you sure we would want to include ES6 arrow functions in Wiki solution? That's the only thing that requires explanation here. Rest looks simple and graspable.

Anyway, barring all that
# LGTM :+1:

File: Algorithm-Diff-Two-Arrays.md [hunk 1]
```diff
@@ -1,93 +1,141 @@
...
+    }
+  }
+
+  return newArr;
+}
+
+diffArray([1, 2, 3, 5], [1, 2, 3, 4, 5]);
+```
+
+<a href="https://repl.it/CLme/0" target="_blank"><kbd><img src="https://i.imgur.com/80UCMbk.png"> <strong>REPL it!</strong> :rocket: </kbd></a>
```
Comment (abhisekp) [inline Algorithm-Diff-Two-Arrays.md]: `target="_blank"` isn't working

Comment (abhisekp) [inline Algorithm-Diff-Two-Arrays.md]: ` target="_blank"` doesn't really work here.

**Pull request merged**